src/python/pants/engine/internals/parametrize.py

```diff
@@ -171,6 +171,24 @@ def get(self, address: Address) -> Target | None:
                 return instance
         return None
 
+    def get_all_compatible_targets(self, address: Address) -> Iterator[Address]:
```

Comment:
"Compatible" may not describe this accurately. Maybe `get_all_superset_targets` or something.